tests/integrations/storage/test_raft_storage.rs

```diff
@@ -365,3 +365,47 @@ fn test_auto_gc() {
         .recv_timeout(Duration::from_millis(300))
         .unwrap_err();
 }
+
+#[test]
+fn test_atomic_basic() {
```

Comment:
Add a case in `integrations/server/kv_service.rs` to cover grpc API

Reply:
Done